mne/decoding/base.py

```diff
@@ -128,6 +130,28 @@ def __repr__(self):  # noqa: D105
         return '%s(%s)' % (class_name, _pprint(self.get_params(deep=False),
                                                offset=len(class_name),),)
 
+    def __getstate__(self):
+        import sklearn
+        if type(self).__module__.startswith('sklearn.'):
+            return dict(self.__dict__.items(),
+                        _sklearn_version=sklearn.__version__)
+        else:
+            return dict(self.__dict__.items())
```

Comment:
Hmm... in that case we might want to keep it the way they do it, even if it's less DRY. Up to you. But if you keep it, please add a comment about things being copied from sklearn (if it's not already there)